packages/terra-disclosure-manager/src/terra-dev-site/doc/disclosure-manager/DisclosingContentWithAHeader.4.doc.mdx

```diff
@@ -0,0 +1,77 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+<Badge />
+
+# Disclosing Content With a Header
+Implementations of the DisclosureManager must render a header containing controls for the various disclosure management actions (close, back, maximize/minimize, etc.). 
+
+The `DisclosureManagerHeaderAdapter` can be used to register header information to the disclosing manager. This can be usd to register a title and a collapsibleMenuView.
+
+## DisclosureManagerHeaderAdapter
+
+The DisclosureManagerHeaderAdapter can be rendered by the disclosed content to inject their own component-specific contents into that header.
+
+`DisclosureManagerHeaderAdapter` Props:
+
+|Prop|Is Required|Description|
```

Comment:
can you not us the props table loader here?

Reply:
These aren't associated with a component so the props table loader doesn't handle it